agent/api/grpc/endpoint_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/ultravioletrs/cocos/agent"
 	"github.com/ultravioletrs/cocos/agent/mocks"
-	config "github.com/ultravioletrs/cocos/pkg/attestation"
+	attestations "github.com/ultravioletrs/cocos/pkg/attestation"
```

Comment:
Why is this called in plural, when library is in singular? Also, do we have to rename this package, or we can use simply it's name `attestation` - i.e. without renaming? And if we have to rename it, maybe we use something shorter, like `att`?

Reply:
Yes, but it is confusing - package name is `attestation`, and then you are importing it with the rename to `attestations`. Either call package `attestations`, or do not rename it in import, use just its original name `attestation` (I think this might be more appropriate here).